micro_dl/utils/masks.py

```diff
@@ -15,14 +15,16 @@ def create_otsu_mask(input_image, style='otsu', thresh_input=0, kernel_size=11):
 
     :param np.array input_image: generate masks from this image
     :param str style: 'otsu', or 'binary'
+    :param iterable thresh_input: default zero, chnages size to input max and min intensity & otsu threshold level of stack middle slice.
```

Comment:
Cosmetic comment: `iterable` is not a built-in type, you might want to consider `Iterable` or `Sequence` from `typing`.

Reply:
Thank you @ziw-liu ! I have fixed it.